### DISCUSSION TITLE
How to remove input focus and hover border color

### DISCUSSION THREAD

**Discussion by rizwan92** [ant-design/ant-design#38797] [General]
### Reproduction link

[reproduction link](https://codesandbox.io/s/ant5-tailwind-test-vt13jb?file=/src/App.js)

### Steps to reproduce

 <Form.Item
            name="email"
            rules={[{ required: true, message: 'Email is required' }]}
          >
            <Input placeholder="Enter Your Email Id" autoFocus />
          </Form.Item>

### What is expected?

custom border color on focus

### What is actually happening?

its applying primary color

| Environment | Info |
| --- | --- |
| antd | 5.0.0 |
| React | 18 |
| System | mac os |
| Browser | chrome |

---
please give a solution for version 5.0.0

**Comment by rizwan92** [comment]
reproduction link updated

**Comment by esu-skoopin** [comment]
You can change the border color for Input components on hover and on focus by using the following CSS:

```
.ant-input:hover {
    border-color: #a4ce88;  /* replace with your desired border color */
}

.ant-input:focus {
    border-color: #a4ce88;  /* replace with your desired border color */
    -webkit-box-shadow: 0 0 0 2px rgba(164, 206, 136, 0.15);  /* replace first three values in rgba() function with the RGB values for your border color */
    box-shadow: 0 0 0 2px rgba(164, 206, 136, 0.15);  /* replace first three values in rgba() function with the RGB values for your border color */
}
```

I created my project using create-react-app, so I just added this CSS into index.css and imported it into my index.js file with `import './index.css';`. Note that this method changes the styles for Input components in the whole project.

[Demo](https://codepen.io/meelybug123/pen/abgxwaZ)

**Comment by wanpan11** [comment]
You can customize the theme using ConfigProvider

![image](https://github.com/user-attachments/assets/97504706-d612-4555-942f-e44a996f04cb)
